_sass/custom/_menu.scss

```diff
@@ -21,19 +21,19 @@
     }
   }
 
-  .navbar li {
+  .nav-item {
     list-style-type: none;
     display: inline-block;
     margin: 0.2rem 1rem;
   }
     
-  .navbar li > a {
+  .nav-item > a {
     text-decoration: none;
     display: inline-block;
     position: relative;
   }
   
-  .navbar li > a::after {
+  .nav-item > .nav-link::after {
```

Comment:
.nav-item class not necessary here, just updating to '.nav-link::after' will work (relates to the 'bootstrap dropdown class fix' in navbar.html).